README.md

```diff
@@ -25,6 +25,24 @@ If using CommonJS modules
 
     const logger = require('@financial-times/n-logger').default;
 
+You may also have different instances of the logger in the same project.  This is useful if you wanted to have different [context](#addcontextmeta) per logger:
+
+```javascript
+  import { getLogger } from '@financial-times/n-logger';
```

Comment:
Given `@financial-times/n-logger` is used for server-side code, better to use...

We're trying to remove Babel from projects that don't need it.

```suggestion
const logger = require('@financial-times/n-logger');
```

Reply:
Good shout, I have corrected it here: https://github.com/Financial-Times/n-logger/pull/86/commits/77514c7a778a07ea45b484348aabb129f694ea27